server/middleware.ts

```diff
@@ -54,6 +54,27 @@ export async function middleware(request: NextRequest) {
     },
   );
 
+  if (process.env.NODE_ENV === "development" && process.env.BYPASS_AUTH === "true") {
+    const dummyEmail: string = process.env.DUMMY_EMAIL!;
+    const dummyPassword:string = process.env.DUMMY_PASSWORD!;
+    const { data, error: signInError } = await supabase.auth.signInWithPassword({
+      email: dummyEmail,
+      password: dummyPassword,
+    })
+
+    if (signInError) {
```

Comment:
is it worth/possible to check the sign in error to see that it is related to invalid credentials?

Reply:
it looks like it should be, but in testing, I don't get a code. I'll open an issue to try to clean this up at some point in the future.